precise/iaf_psc_alpha_ps.h

```diff
@@ -172,6 +172,13 @@ class iaf_psc_alpha_ps : public Archiving_Node
   void get_status( DictionaryDatum& ) const;
   void set_status( const DictionaryDatum& );
 
+  /**
+   * Difference between threshold and membrane potential for given time step.
+   * @param   double time step
+   * @returns difference between updated membrane potential and threshold
+   */
```

Comment:
No need to repeat doxygen comment when overriding a virtual method.

Reply:
I have removed the virtual method.